src/main/java/com/thealgorithms/divideandconquer/ConvexHull.java

```diff
@@ -0,0 +1,228 @@
+package com.thealgorithms.divideandconquer;
```

Comment:
It seems that this PR is closely related to the existing [GrahamScan implementation](https://github.com/TheAlgorithms/Java/blob/master/src/main/java/com/thealgorithms/geometry/GrahamScan.java).

The Point class (or record) from the GrahamScan class can be reused here, and it might be beneficial to review both implementations to identify reusable components. This would help avoid redundancy and improve maintainability. It might also be worth considering merging or unifying the logic in some way to make the code more efficient and consistent across both classes.

Reply:
Done!